lib/streamlit/runtime/memory_session_storage.py

```diff
@@ -55,7 +55,7 @@ def __init__(
             inaccessible and will be removed eventually.
```

Comment:
nit: may want to tweak this docstring since we now have nicer guarantees on entry removal